.craft.yml

```diff
@@ -3,6 +3,29 @@ changelogPolicy: auto
 preReleaseCommand: bash scripts/craft-pre-release.sh
 targets:
   - name: npm
+  - name: brew
+    tap: getsentry/tools
+    template: >
+      require 'language/node'
+
+      class SentryWizard < Formula
+        desc "The Sentry Wizard helps you set up your projects with Sentry"
+        homepage "https://github.com/getsentry/sentry-wizard"
+        url "https://registry.npmjs.org/@sentry/wizard/-/wizard-{{version}}.tgz"
+        sha256 "{{checksums.sentry-wizard-vlatest__tgz}}"
+        version "{{version}}"
+        license "MIT"
+
+        depends_on "node"
+
+        def install
+          system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+          bin.install_symlink Dir["#{libexec}/bin/*"]
```

Comment:
I lack a lot of context. Do you mind quickly explaining what this does?

Reply:
Right, sorry, you can take a look at this PR which adds a new formula for sentry-wizard https://github.com/getsentry/homebrew-tools/pull/6 and the supporting Homebrew guide when publishing Node formulas https://docs.brew.sh/Node-for-Formula-Authors

Reply:
Citing the docs above

> Node modules should be installed to libexec. This prevents the Node modules from contaminating the global node_modules, which is important so that npm doesn’t try to manage Homebrew-installed Node modules.

Reply:
Makes sense. Thanks!